src/arrays/index.md

```diff
@@ -129,13 +128,13 @@ libaverages.a(std-9a66b6a343d52844.0.o): In function `std::sys::imp::mutex::{{im
 clang: error: linker command failed with exit code 1 (use -v to see invocation)
 ```
 
-Oops! When clang tried to compile our `libaverages.a` library and `main.c` into
-one executable it wasn't able to find a bunch of symbols.
+Oops! When clang tried to compile the libraries `libaverages.a` and `main.c`
```

Comment:
I'm not sure if "the libraries" is the right wording here. Technically only `libaverages.a` is a library while `main.c` is a program...

What about going with the original paragraph and just removing the "our" on the first line?